frontend/src/scenes/plugins/plugin/PluginLogs.tsx

```diff
@@ -78,6 +78,16 @@ export function PluginLogs({ pluginConfigId }: PluginLogsProps): JSX.Element {
                     allowClear
                 />
             </Row>
+            <Row>
+                <span>
+                    Show logs of type:&nbsp;
+                    <Checkbox.Group
+                        options={Object.values(PluginLogEntryType)}
+                        defaultValue={Object.values(PluginLogEntryType)}
```

Comment:
I think it's preferred (less surprises) to directly control any kind of input with `value={pluginLogsTypes}`, instead of assuming only unidirectional data flow via `defaultValue` + `onChange`. Currently if you change the reducer in the logic, the checkboxes won't update.

It won't be a problem now with the current implementation, but if possible, let's wire this up.

Reply:
I'm not sure I've completely understood this point... I had to add listeners that trigger actions 🤔